backend/spec/system/sign_in_spec.rb

```diff
@@ -15,6 +15,8 @@
     it { expect(page).to have_current_path('/') }
 
     it { expect(page).to have_text('Signed in successfully') }
+
+    it { expect(page).to have_link('Logout', href: destroy_user_session_path) }
   end
```

Comment:
Would be nice to have a spec that checks user is logged out after clicking on this button.